Assets/js/script.js

```diff
@@ -0,0 +1,178 @@
+// My Criteria Arrays
+const lower = [
+  "a",
+  "b",
+  "c",
+  "d",
+  "e",
+  "f",
+  "g",
+  "h",
+  "i",
+  "j",
+  "k",
+  "l",
+  "m",
+  "n",
+  "o",
+  "p",
+  "q",
+  "r",
+  "s",
+  "t",
+  "u",
+  "v",
+  "w",
+  "x",
+  "y",
+  "z",
+];
+const upper = [
+  "A",
+  "B",
+  "C",
+  "D",
+  "E",
+  "F",
+  "G",
+  "H",
+  "I",
+  "J",
+  "K",
+  "L",
+  "M",
+  "N",
+  "O",
+  "P",
+  "Q",
+  "R",
+  "S",
+  "T",
+  "U",
+  "V",
+  "W",
+  "X",
+  "Y",
+  "Z",
+];
+const number = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"];
+const special = [
+  " ",
+  "!",
+  "\\",
+  "#",
+  "$",
+  "%",
+  "&",
+  "'",
+  "(",
+  ")",
+  "*",
+  "+",
+  ",",
+  "-",
+  ".",
+  "/",
+  ":",
+  ";",
+  "<",
+  "=",
+  ">",
+  "?",
+  "@",
+  "[",
+  "]",
+  "^",
+  "_",
+  "`",
+  "{",
+  "|",
+  "}",
+  "~",
+];
+
+const myPasswordFail = "Try again!";
+
+// Assignment Code
+var generateBtn = document.querySelector("#generate");
+
+function generatePassword() {
+  combinedCriteria = []; //my empty array to fill up
+  // example of user saying yes to lower and upper and no to rest below
+  // make this, for each of the proimpt I have gotten from user, i need to adjust this array(add more to it) if the user has said yes to any of them.
+  // if user has said yes to prompts, it gets added to array.
+  // concanate will combine the two strings together.
+  // combinedCriteria = combinedCriteria.concat(upper); //can get rid of this when sorted above array.
```

Comment:
Please delete any commented out code to make the code clearer and easier to read